catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -310,8 +310,8 @@ private MetacardValidationReport generateReport(Metacard metacard)
         MetacardValidationReportImpl report = new MetacardValidationReportImpl();
         Set<String> attributes = ImmutableSet.of("metadata");
         String metadata = metacard.getMetadata();
-        boolean canBeValidated = StringUtils.isNotEmpty(metadata) || namespace == null
-                || namespace.equals(XMLUtils.getRootNamespace(metadata));
+        boolean canBeValidated = StringUtils.isNotEmpty(metadata) && namespace != null
```

Comment:
@shaundmorris comment about a unit test stands. Even if it is not submitted today, a ticket should be opened to create one.